src/scenarios/shared/sdk_iteration_setup.py

```diff
@@ -0,0 +1,20 @@
+import os
+import shutil
+from shared import const
+from dotnet import shutdown_server
+from performance.logger import setup_loggers
+from logging import getLogger
+
+
+def main():
+    setup_loggers(True)
+    shutdown_server(verbose=True)
+    getLogger().info("Removing project directory...") # set up logger would create new log files, which is not necessary
+    if os.path.isdir(const.TMPDIR):
+        shutil.rmtree(const.TMPDIR)
+    getLogger().info("Copying clean project directory...")
+    shutil.copytree(const.APPDIR, const.TMPDIR)
```

Comment:
This code is duplicated in `precommands.py`. It's basically the same as the `backup` function. Given that the harness will run this every time, the `tmp` directory will be created as part of `pre.py` and then again each iteration. There's a wasted step there. It's fine if SDK tests don't have `pre.py` invocations if they don't need them.

Reply:
The current test order is build(no change) runs after clean build. However, if we flip the order or run only the build(no change) scenario, pre.py will still be needed.

Reply:
Okay.